@navikt/core/react/src/overlays/floating-menu/parts/RovingFocus.tsx

```diff
@@ -7,7 +7,12 @@ import { DescendantsManager } from "../../../util/hooks/descendants/descendant";
 interface RovingFocusProps
   extends Omit<React.HTMLAttributes<HTMLDivElement>, "tabIndex"> {
   asChild?: boolean;
-  descendants: DescendantsManager<HTMLDivElement, object>;
+  descendants: DescendantsManager<
+    HTMLDivElement,
+    {
+      closeMenu: () => void;
```

Comment:
It feels wrong that `RovingFocus` should know about menus 🤔
I think we can/should add a type parameter here, but don't know how to do it when it the component uses `forwardRef` 🤔 We could consider removing `forwardRef` and possibly add it as a regular prop instead.

```ts
interface RovingFocusProps<T extends Record<string, any> = object>
  extends Omit<React.HTMLAttributes<HTMLDivElement>, "tabIndex"> {
  asChild?: boolean;
  descendants: DescendantsManager<HTMLDivElement, T>;
  onEntryFocus?: (event: Event) => void;
}
```

Reply:
Got it to work with forwardRef 
```
interface RovingFocusProps<T extends Record<string, any> = object>
  extends Omit<React.HTMLAttributes<HTMLDivElement>, "tabIndex"> {
  asChild?: boolean;
  descendants: DescendantsManager<HTMLDivElement, T>;
  onEntryFocus?: (event: Event) => void;
}

const ENTRY_FOCUS = "rovingFocusGroup.onEntryFocus";
const EVENT_OPTIONS = { bubbles: false, cancelable: true };

const RovingFocus = forwardRef<HTMLDivElement, RovingFocusProps>(
  <T extends Record<string, any> = object>(
    {
      children,
      asChild,
      descendants,
      onKeyDown,
      onEntryFocus,
      onMouseDown,
      onFocus,
      ...rest
    }: RovingFocusProps<T>,
    ref: React.Ref<HTMLDivElement>,
  ) => {
```

But still getting errors since `Record<string, any>` is not generic enough
![Screenshot 2024-10-07 at 10 34 24](https://github.com/user-attachments/assets/0d5746d3-9d26-4630-8dc3-fd4b4d0bfc42)

But since the props are not relevant here, i could live with just
```
descendants: DescendantsManager<HTMLDivElement, any>;
```

Reply:
I think the problem is that the type param is missing from `forwardRef<HTMLDivElement, RovingFocusProps>`, but I guess `any` would be fine here.